parley/src/layout/data.rs

```diff
@@ -199,6 +200,9 @@ pub(crate) struct LayoutData<B: Brush> {
     pub(crate) fonts: Vec<Font>,
     pub(crate) coords: Vec<i16>,
 
+    // Lazily calculated values
+    content_widths: OnceCell<ContentWidths>,
```

Comment:
It would be good to add the changes to the changelog, and mention as breaking change Layout is no longer sync.